ui/plugins/controller.js

```diff
@@ -141,6 +154,32 @@ treeherder.controller('PluginCtrl', [
                           }
                           return result;
                         }, []);
+                        if ($scope.job_ids !== null) {
+                            // Due to we still need to keep the GraphServer link in here, right now the only
+                            // way to get the talos result is from jobInfoArtifact. Anyway, we can
+                            // make talos just pass the result directly if we don't need Graph link anymore.
+                            $scope.job_details.forEach(function(job) {
+                                if (job.content_type == 'TalosResult') {
+                                    $scope.perf_job_deatils = [];
+                                    _.forEach(job.value.graphserver, function(test, testName) {
+                                        _.forEach($scope.job_ids, function(perfdata, key) {
+                                            // we only compare 1 digit after the decimal point because we may meet
+                                            // compare 14.10 with 14.12 and actually they point to the same job.
+                                            if (perfdata[0].value.toFixed(1) == Number(test.result).toFixed(1)) {
+                                                var data = {};
+                                                if (_.some($scope.perf_job_deatils, 'name', testName) === false) {
+                                                    data['name'] = testName;
+                                                    data['value'] = test.result;
+                                                    data['url'] = generateUrlForPerf($scope.repoName,
+                                                        perfdata[0].job_id, key, perfdata[0].result_set_id);
+                                                    $scope.perf_job_deatils.push(data);
+                                                }
```

Comment:
Yeah, don't worry about that for now. Just print out the raw value in bytes. We can write a formatter for this type of data later, when we encode it.